frontend/src/components/shared/ColorPicker/ColorPicker.tsx

```diff
@@ -104,14 +104,16 @@ class ColorPicker extends React.PureComponent<Props, State> {
     }
     return (
       <StyledColorPicker data-testid="stColorPicker" style={style}>
-        <StyledWidgetLabel>
-          {label}
-          {help && (
-            <StyledWidgetLabelHelpInline>
-              <TooltipIcon content={help} placement={Placement.TOP_RIGHT} />
-            </StyledWidgetLabelHelpInline>
-          )}
-        </StyledWidgetLabel>
+        {label ? (
```

Comment:
Should this use the new `WidgetLabel` wrapper?

Same comment for `Selectbox.tsx`, and `Radio.tsx`

Reply:
Hmmm... I had a good reaso not to do this but now I can't remember why. Something related to how these aren't BaseWidgets. But that sounds silly, so I've updated them for you!